themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+
+While the Puluminaries themselves have been active in the community for quite some time, we officially launched the program at [PulumiUP](https://pulumi.com/pulumi-up) in 2022, and it is our pleasure to introduce the first cohort of members to the program!
+
+## Meet The Puluminaries!
+
+### Engin Diri
+
+<img src="engin-diri.png" width="200" height="200" alt="Engin Diri"/>
+<a data-track="twitter" href="https://twitter.com/_ediri"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/dirien"><i class="fab fa-github"></i></a>
+
+Engin's excitement for Pulumi is not possible to be measured using current science. He will always share his thoughts on new developements in the Pulumi project, as well as join in for fun and discussion in various PulumiVision livestreams!
+
+### Joshua Studt
+
+<img src="joshua-studt.png" width="200" height="200" alt="Joshua Studt"/>
+<a data-track="github" href="https://github.com/orionstudt"><i class="fab fa-github"></i></a>
+
+Joshua is a longtime member of the Pulumi community, and he contributed to the implementation of Automation API for C#.
+
+### Kat Morgan
+
+<img src="kat-morgan.png" width="200" height="200" alt="Kat Morgan"/>
+<a data-track="twitter" href="https://twitter.com/usrbinkat"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/usrbinkat"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/usrbinkat"><i class="fab fa-linkedin"></i></a>
+
+Kat has been a helpful member of the Pulumi community - not only providing feedback on real-world use of Pulumi, but also sharing her knowledge with [workshops and webinars](https://konghq.com/webinars/devmyops-deploy-kong-with-pulumi?utm_souce=pulumi&utm_medium=pulumi)
+
+### Komal Ali
+
+<img src="komal-ali.png" width="200" height="200" alt="Komal Ali"/>
+<a data-track="twitter" href="https://twitter.com/zwitkali"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/komalali"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/komal-ali"><i class="fab fa-linkedin"></i></a>
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers -  they were ever-present in making our Python SDK better and championing [Automation API](https://www.pulumi.com/docs/guides/automation-api/)!
+
+### Paul Hicks
+
+<img src="paul-hicks.png" width="200" height="200" alt="Paul Hicks"/>
+<a data-track="github" href="https://github.com/tenwit"><i class="fab fa-github"></i></a>
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
```

Comment:
Prolific helper sounds light. I think we want to give a sense of the impressive knowledge that he brings to the table. 

Why don't we say "Paul seems to have an answer for everything. He is ever present in the Community answering the simple getting started questions to more complex cloud architecture questions" or something like that and as along as that is actually true.

Reply:
I updated this and added your suggestions!